qiskit_addon_sqd/fermion.py

```diff
@@ -39,28 +41,36 @@
 
 
 def solve_fermion(
-    addresses: tuple[np.ndarray, np.ndarray],
+    bitstring_matrix: tuple[np.ndarray, np.ndarray] | np.ndarray,
```

Comment:
The docstring doesn't explain what it means to pass a `tuple[np.ndarray, np.ndarray]`.

Also, I still think it would be useful to support passing the bitstrings as a list of integers rather than a bit array. I won't insist on adding it now, but if you think it's a good idea, then we should give this parameter a more generic name, such as `bitstrings`.

Reply:
Well, the `bitstring_matrix` is like the main data structure of the entire repo. It facilitates all the various numpy stuff we want to do on the bitstrings. I wouldn't want to change the name. I don't see what passing the determinants as integers brings to the table here. If you're doing SQD, you have a bitstring matrix, and if you use our batching function, you also have a bitstring matrix handy.

Reply:
Ok, now that I've thought about it for two minutes ... maybe it's worthwhile to change the arg name in the solver functions only and relax the input type for the chem solvers to include the base-10 representation.

Reply:
Actually, I take what I said back. You're right that `bitstring_matrix` is the main data structure, and it's the appropriate format for doing configuration recovery. I think it's okay to take only a bitstring matrix as input everywhere, and simply rely on helper functions to convert between bitstring formats as needed.